assets/locales/en.json

```diff
@@ -9,6 +9,17 @@
       "counterVotes": "No Votes",
       "requestedAmount": "Requested Amount",
       "votes": "Yes Votes"
+    },
+    "filterResponse": {
+      "noSearch": {
+        "heading": "No proposals <span class='text-primary'>yet :(</span>",
```

Comment:
issue(blocking): Let's avoid using html and css classes in our locales.
suggestion: Move html into the component